festim/hydrogen_transport_problem.py

```diff
@@ -385,51 +396,56 @@ def assign_functions_to_species(self):
             spe.prev_solution = sub_prev_solution[idx]
             spe.test_function = sub_test_functions[idx]
 
-    def define_markers_and_measures(self):
-        """Defines the markers and measures of the model"""
+    def define_meshtags_and_measures(self):
```

Comment:
Note: we need to think of a way for users to provide their own fenics mesh with their own meshtags.

Here we have only two options: tags from XDMF, or 1D mesh. What about users providing a cube mesh but tagging the surfaces themselves?